scripts/commands/update-translations.js

```diff
@@ -44,13 +44,19 @@ async function verifyLangData( langData ) {
 
 export default async function updateTranslations() {
 	const git = gitFactory();
+	const wpPath = 'vendor/bin/wp';
 
 	try {
+		if ( ! fs.existsSync( wpPath ) ) {
+			warning( 'wp not found, running composer install...' );
+			await execPromise( 'composer install' );
+		}
+
 		info( 'Executing wp i18n make-pot command...' );
 
 		// If the `wp i18n make-pot` command generates an error, an exception is thrown
 		const makePotResult = await execPromise(
```

Comment:
One more minor nit. It doesn't look like we're using `makePotResult` anywhere so we can probably delete that variable.

Reply:
Done